ministryofjustice/laa-review-criminal-legal-aid
# Rename column headings on closed application tables

## Description of change
Change column headers from date reviewed and reviewed by to date closed and closed by respectively

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-171

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
<img width="1505" alt="DateReviewed:ReviewedBy" src="https://user-images.githubusercontent.com/106679087/228878493-4e10263a-e733-47a2-98cc-3c16f3172caf.png">

### After changes:
<img width="1488" alt="DateClosed:ClosedBy" src="https://user-images.githubusercontent.com/106679087/228878530-0accf675-a22c-4f8c-938a-21ae35f79767.png">

## How to manually test the feature
- User logs into application
- User clicks on Closed applications button
- User is guided to the Closed applications page
- User should see a list of closed applications in the table similar to the screenshots as attached above
- User should see that date reviewed has been changed to date closed. User can click on Date closed to sort Closed application by oldest to newest and newest to oldest.
- User should see that reviewed by has been changed to Closed by with their showing in the column below.
